packages/elements/src/overlay-menu/index.ts

```diff
@@ -774,26 +797,36 @@ export class OverlayMenu extends Overlay {
 
   /**
    * Highlight next or previous highlightable element if present
-   * @param direction -1 - up/next; 1 - down/previous
+   * @param direction previous, next, first or last focusable element
    * @param [circular=false] Set to true to have circular navigation over items
    * @return {void}
    */
-  private focusElement (direction: number, circular = false): void {
+  private focusElement (direction: Navigation, circular = false): void {
     const menuHighlightedItem = this.menuHighlightedItem;
     const children = this.items;
 
     const idx = menuHighlightedItem ? children.indexOf(menuHighlightedItem) : -1;
-
     let focusElement;
-    if (direction === 1) {
-      focusElement = idx === -1 ? children[0] : children[idx + 1];
-    }
-    else {
-      focusElement = idx === -1 ? children[children.length - 1] : children[idx - 1];
-    }
 
+    switch (direction) {
+      case Navigation.PREVIOUS:
+        focusElement = idx === -1 ? children[children.length - 1] : children[idx - 1];
+        break;
+      case Navigation.NEXT:
+        focusElement = idx === -1 ? children[0] : children[idx + 1];
+        break;
+      case Navigation.FIRST:
+        focusElement = children[children.length - 1];
+        break;
+      case Navigation.LAST:
+        focusElement = children[0];
```

Comment:
This is wrong, switch `FIRST` to `LAST`.
```suggestion
        focusElement = children[children.length - 1];
```

Reply:
fixed